services/src/pro/datasets/external/sentinel_s2_l2a_cogs.rs

```diff
@@ -585,6 +585,7 @@ impl MetaData<GdalLoadingInfo, RasterResultDescriptor, RasterQueryRectangle>
             measurement: Measurement::Unitless,
             time: None,
             bbox: None,
+            resolution: None,
```

Comment:
Don't we have one for sentinel? It should be 10m? Or do we get this from metadata?

Reply:
I don't know. We should maybe investigate this in a separate task.

Reply:
Can you add a TODO then?

Reply:
ecb4ae36da7c1cb88d8d66ecaee478c080493c56